src/ecdsa/keys.py

```diff
@@ -990,9 +1058,11 @@ def to_pem(self, point_encoding="uncompressed"):
         Only the named curve format is supported.
         The public key will be included in generated string.
 
-        The PEM header will specify ``BEGIN EC PRIVATE KEY``
+        The PEM header will specify ``BEGIN EC PRIVATE KEY`` or
+        ``BEGIN PRIVATE KEY``, depending on the desired format.
 
         :param str point_encoding: format to use for encoding public point
+        :param str format: either `sslay` or `pkcs8`
```

Comment:
```
, `ssleay` by default
```

Reply:
btw, if you want `pkcs8` in monospace font, it should use double backticks:
```
``ssleay`` or ``pkcs8``
```

See https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html